src/i18n/messages-fr.po

```diff
@@ -0,0 +1,875 @@
+
+msgid "auto"
+msgstr "auto"
+
+msgid "Send stereo audio"
+msgstr "Envoyer de l'audio stéréo"
+
+msgid ""
+"Sending stereo audio disables echo cancellation. Enable only if you have "
+"stereo input."
+msgstr ""
+"Envoyer de l'audio en stéréo désactive la suppression de l'écho."
+"Utilisez le uniquement si vous avez un micro stéréo."
+
+msgid "Detect CPU over use"
+msgstr "Détecter un usage processeur excessif"
+
+msgid "Automatically reduces video quality as needed."
+msgstr "Réduire automatiquement la qualité vidéo si besoin."
+
+msgid "Optimize for high resolution video"
+msgstr "Optimiser la vidéo pour la haute résolution"
+
+msgid "Reduce video noise"
+msgstr "R2duire le bruit vidéo"
+
+msgid "Prefer VP9 video codec"
+msgstr "Utiliser le codec VP9 en priorité"
+
+msgid "Enable experiments"
+msgstr "Activer les réglages experimentaux"
+
+msgid "Show advanced settings"
+msgstr "Montrer les réglages avancés"
+
+msgid "Hide advanced settings"
+msgstr "Masquer les réglage avancés"
+
+msgid "Remember settings"
+msgstr "Se souvenir des réglages"
+
+msgid ""
+"Your ID will still be kept - press the log out button above to delete the"
+" ID."
+msgstr ""
+"Votre ID sera conservé - appuyer sur le bouton de déconnection pour l'effacer"
+""
+
+msgid "Room link"
+msgstr "Lien vers la salle"
+
+msgid "Invite by Email"
+msgstr "Inviter par email"
+
+msgid "Invite with Facebook"
+msgstr "Inviter par Facebook"
+
+msgid "Invite with Twitter"
+msgstr "Inviter par Twitter"
+
+msgid "Invite with Google Plus"
+msgstr "Inviter par Google Plus"
+
+msgid "Invite with XING"
+msgstr "Inviter par XING"
+
+msgid "Initializing"
+msgstr "En cours d'initialisation"
+
+msgid "Online"
+msgstr "En ligne"
+
+msgid "Calling"
+msgstr "Appel en cours"
+
+msgid "Hangup"
+msgstr "Raccrocher"
+
+msgid "In call with"
+msgstr "En communication avec "
+
+msgid "Conference with"
+msgstr "En conférence avec"
+
+msgid "Your are offline"
+msgstr "Vous êtes hors ligne"
+
+msgid "Go online"
+msgstr "Passer en ligne"
+
+msgid "Connection interrupted"
+msgstr "Connection interrompue"
+
+msgid "An error occured"
+msgstr "Une erreur est survenue"
+
+msgid "Incoming call"
+msgstr "Appel entrant"
+
+msgid "from"
+msgstr "de"
+
+msgid "Accept call"
+msgstr "Accepter l'appel"
+
+msgid "Waiting for camera/microphone access"
+msgstr "En attente de l'accès à la webcam et au micro"
+
+msgid "Your audio level"
+msgstr "Niveau de votre audio"
+
+msgid "Checking camera and microphone access."
+msgstr "Vérification de l'accès à la webcam et au micro."
+
+msgid "Please allow access to your camera and microphone."
+msgstr "Merci d'autoriser l'accès à la webcam et au micro."
+
+msgid "Camera / microphone access required."
+msgstr "Accès requis à la webcam et au micro."
+
+msgid ""
+"Please check your browser settings and allow camera and microphone access"
+" for this site."
+msgstr ""
+"Merci de verifier les reglages de votre navigateur et pour autoriser"
+"l'accès au micro et a la webcam pour ce site."
```

Comment:
Missing leading whitespace (to have a separator between this and the line before).